<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Countdown</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="timeHorizontalLayout">
        <item>
         <widget class="QSpinBox" name="hoursSpinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="hoursLabel">
          <property name="text">
           <string>hours</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="minutesSpinBox">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="minutesLabel">
          <property name="text">
           <string>minutes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="secondsSpinBox">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="secondsLabel">
          <property name="text">
           <string>seconds</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="fileLabel">
        <property name="text">
         <string>File (path) to open when timer ends:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="controlHorizontalLayout">
        <item>
         <widget class="QLineEdit" name="fileLineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browsePushButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>7</number>
        </property>
        <item>
         <widget class="QPushButton" name="startPushButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pausePushButton">
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopPushButton">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
